src/routes/game/components/CannotCounterDialog.vue

```diff
@@ -46,9 +46,19 @@
     </template>
 
     <template #actions>
+      <v-btn
+        data-cy="cannot-counter-counter"
```

Comment:
This selector may not be necessary because we're not using it in testing atm, but if we did want to assert the existence and disabled state of this button, this is how we could select it